docs/spec.md

```diff
@@ -6312,6 +6330,19 @@ function returns `true`. If `x` is not a tensor, returns `None`.
 tensors and returns `num_results` slices of `x` along the axis `axis`.
 If `x` is not a tensor or `dim(x, axis) % num_results != 0`, returns `None`.
 
+* `reduce_implicit_convert(x: Value, source_type: Type, destination_type: Type)
+-> Value` is defined on tensors and returns the converted value of `x` based on
+the `source_type` and `destination_type` as follows:
+
+```python
+def reduce_implicit_convert(x: Value, source_type: Type, destination_type: Type) -> Value:
```

Comment:
Am I right in thinking that `type(x)` is equal to `source_type` for all calls to `reduce_implicit_convert`? If yes, we probably don't need to pass `source_type` explicitly?

Reply:
Thanks for pointing this. Yes, we do not need the extra argument.